### DISCUSSION TITLE
Shouldn't the sum_vars function handle the character vector of the column name instead of using !!!rlang::syms()

### DISCUSSION THREAD

**Discussion by lanawwas** [impact-initiatives-hppu/humind#411] [General]
There is an error currently in the unit-test due to the use of `!!!rlang::syms()` in the code. While the `sum_vars` can handle the column names as a character vector

I propose to change the following, by replace the `!!!rlang::sysms()` with character vector in [add_comp_prot,R#L133](https://github.com/impact-initiatives-hppu/humind/blob/02a748112bd64760e1ae69d4bbfd7df565e44a5a/R/add_comp_prot.R#L113) : 

```r
# now sum across with the following threshold: 11 and above = 5, 8 to 10 = 4, 5 to 7 = 3, 2 to 4 = 2, 1 = 1, NA = NA
df <- sum_vars(
  df,
  c("comp_prot_score_concern_freq_cope", "comp_prot_score_concern_freq_displaced", "comp_prot_score_concern_hh_freq_kidnapping", "comp_prot_score_concern_hh_freq_discrimination"),
  "comp_prot_score_concern",
  na_rm = FALSE,
  imputation = "none"
)
```

@gnoblet

**Comment by gnoblet** [comment]
Perfect, thanks

**Comment by lanawwas** [comment]
Solved in 59efc98
